cfgov/templates/welcome.html

```diff
@@ -3,14 +3,42 @@
 
 {% block furniture %}
     <div class="content-wrapper">
+
         <h1>{% trans 'Welcome' %} <br/>
-        You've logged in to the consumerfinance.gov editorial tools.	
+        {% if 'build' in request.site.hostname %}
```

Comment:
I kind of agree with @rosskarchner on L22. Why not just expose this everywhere, and then we don't have to worry about special-casing build (especially as we've been exploring options other than build for the Wagtail playground)?